### ISSUE TITLE
SSIM is much lower than the results in the paper

### ISSUE THREAD

**Issue by wushuang01** [KupynOrest/DeblurGAN#124]
I trained the GoPro dataset, but only got the 86.07 ssmi on the GoPro test data, which is much lower than the 95.4 in the paper. Has anyone encountered a similar problem?

**Comment by JaihyunKoh** [comment]
I also trained deblurGAN model with the GoPro dataset and defalt hyperparameters. My result is SSIM 0.8151 for 640x480 test set images. What's wrong? Anyone who can explain to us? Thanks.

**Comment by YRunner** [comment]
Do you fix it? I also want to know how do you get the SSIM, just run the test.py?

**Comment by YRunner** [comment]
I test with GoPro dataset 1080×720, My result is SSIM 0.799  PSNR 25.5 much lower than the paper

**Comment by 1z1y** [comment]
> 

Now I also have the problems with the low ssim and psnr , could you tell me  how did you solve this problems ?

**Comment by hhrsn** [comment]
> > 
> 
> 现在我也有低ssim和psnr的问题，你能告诉我你是如何解决这个问题的吗？

Have you reproduced the data from the original article, I'm also quite far from the original article
